rdd/src/main/scala/ast/RDDSource.scala

```diff
@@ -0,0 +1,15 @@
+package maml.rdd.ast
+
+import maml.ast._
+
+import geotrellis.spark._
+
+
+case class SpatialRDDLiteral(value: TileLayerRDD[SpatialKey]) extends Source {
+  val id = value.toString
+  val kind = MamlKind.Tile
+}
+
+case class SpatialRDDSource(id: String) extends UnboundSource {
+  val kind = MamlKind.Tile
+}
```

Comment:
@moradology Do you think we need this case class or can we use `TileSource` in all cases where we have a variable of `MamlKind.Tile` ?